docs/wiki/Client-control-API.mediawiki

```diff
@@ -66,7 +66,7 @@ The response will contain a help message based on the arguments.
 
 ===Version===
 
-Print the DEX client rpcserver version. Accepts no arguments
+Print the Bison Wallet rpcserver version. Accepts no arguments
```

Comment:
In some places we no longer refer to `the Bison Wallet` but `Bison Wallet`, intentional? IF just `Bison Wallet` sounds better we should have that here and every other place below where we do `The Bison Wallet` I think.

Reply:
The name is certainly **Bison Wallet** and not **The Bison Wallet**, but most of these are OK. In the statement `the Bison Wallet rpcserver version` or `the Bison Wallet password`, **the** points to **version** or **password**, and **Bison Wallet** just specifies **which** password or version.